public/register.html

```diff
@@ -1,50 +1,54 @@
-<!-- 
-    Team Teal: Andrew Cook, Cheryl Moser, Petar Spasic
-    Date:
-    AD320 Final Project
- -->
-
-<!DOCTYPE html>
-
-<html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <link rel="stylesheet" href="styles.css">
-        <script src="index.js"></script>
-        <link rel="preconnect" href="https://fonts.googleapis.com">
-        <link rel="preconnect" href="https://fonts.gstatic.com">
-        <link href="https://fonts.googleapis.com/css2?family=Dosis:wght@400&display=swap" rel="stylesheet">
-        <title>Go Outside!</title>
-    </head>
-
-    <body>
-        <section id = "top">
-            <hgroup>
-                <h1>
-                    Welcome to Go Outside Co.
-                    <br>Equipment Reservations
-                </h1>
-                <nav><a href="index.html">reservations home</a> || login</nav>
-            </hgroup>
-        </section>
-
-        <section id="content">
-            <section id = "result">
-                <form id="create_user" action="/api/user/register" method="post">
-                    <div>
-                        <label for="username">Enter a username: </label>
-                        <input type="email" name="email" required />
-                    </div><br>
-                    <div>
-                        <label for="password">Enter a password:  </label>
-                        <input type="password" name="password" required /><br><br>
-                        <input type="submit" value="submit"><br><br>
-                        <span></span>
-                    </div>
-                </form>
-            </section>
-        </section>
-
-        <footer>Website by Team Teal  -  AD320 Fall 2023</footer>
-    </body>
+<!-- 
+    Team Teal: Andrew Cook, Cheryl Moser, Petar Spasic
+    Date:
+    AD320 Final Project
+ -->
+
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="styles.css">
+        <script src="index.js"></script>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Dosis:wght@400&display=swap" rel="stylesheet">
+        <title>Go Outside!</title>
+    </head>
+
+    <body>
+        <section id = "top">
+            <hgroup>
+                <h1>
+                    Welcome to Go Outside Co.
+                    <br>Equipment Reservations
+                </h1>
+                <nav><a href="index.html">reservations home</a> || <a href="login.html" id="log">log in</a></nav>
+            </hgroup>
+        </section>
+
+        <section id="content">
+            <section id = "result">
+                <form id="create_user" action="/api/user/register" method="post">
+                    <div>

```

Comment:
If we are using email/password can this be removed?

Reply:
Remove the ability to add new users to the database?

Reply:
The part where we ask for a username.